Sources/SwiftCrossUI/Modifiers/CSSModifierView.swift

```diff
@@ -0,0 +1,31 @@
+import Gtk
+
+/// A modifier view on which you can apply CSS
+struct CSSModifierView<Child: View>: View {
+    var body: ViewContent1<Child>
+
+    var properties: [CSSProperty]
+
+    var clear: Bool
+
+    var modifierName: String
+
+    /// - Parameter clear: Whether to clear all properties before applying new. By default the properties
+    ///                    are additive and it does not remove css properties previously set.
+    internal init(_ child: Child, properties: [CSSProperty], clear: Bool = false, modifierName: String = #function) {
+        self.body = ViewContent1(child)
+        self.properties = properties
+        self.clear = clear
+        self.modifierName = modifierName
+    }
+
+    func asWidget(_ children: ViewGraphNodeChildren1<Child>) -> GtkModifierBox {
+        let widget = GtkModifierBox().debugName(Self.self)
```

Comment:
Should we pass `modifierName` to `debugName` (potentially as some optional 3rd context parameter) to make it clear which modifier was used to create the CSSModifierView?

Reply:
Yeah I was looking into if we can't pass more variables to the inspector side but I can't find an alternative to `Widget.name`. We should add an "id" param to debugName function. I will not have time to update this PR until next week though.

Reply:
Yeah all good 👍 No rush